lib/LaTeXML/Package/TeX.pool.ltxml

```diff
@@ -6769,16 +6823,22 @@ DefConstructor('\lx@text@nounicode DefToken',
 #
 # To get a "proper id", we'll use a temporary label-like attribute (_xmkey)
 # and establish an id and idref later.
-DefConstructor('\@XMArg{}{}', "<ltx:XMArg _xmkey='#1'>#2</ltx:XMArg>",
+DefConstructor('\lx@xmarg{}{}', "<ltx:XMArg _xmkey='#1'>#2</ltx:XMArg>",
   reversion   => '#2',
   afterDigest => sub {
+    my ($stomach, $whatsit) = @_;
+    my ($xmid,    $arg)     = $whatsit->getArgs();
+    $xmid = ToString($xmid);
+    my $xmargs = $PENDING_DUAL_XMARGS[-1];
+    $$xmargs{$xmid} = $arg;
```

Comment:
Is `$xmargs` always defined here? perl die if not.

Reply:
In particular, I am reminded one could use XMArg in XMath independently of duals, e.g. for fraction-like constructs.